tests/sdk/query/test_validate_pums.py

```diff
@@ -38,6 +39,7 @@
 bad_files = [f for f in validate_files if "_fail" in f]
 
 metadata = CollectionMetadata.from_file(meta_path)
+metadata.compare = PostgresNameCompare()
 
```

Comment:
This was necessary after fixing the issue in line 257 of ast.py.  We have a unit test to make sure we get a validation error on `SELECT SUM(Age + Income) FROM PUMS.PUMS`.  This was passing before the bug fix, but failed after the bug fix, because it was using `PandasReader`, which is case sensitive.  So, this unit test should have been failing.  Attaching the `PostgresNameCompare` switches the test to be case-insensitive, so the tests succeed.